string im_name
---
sensor_msgs/Image img
